frontend/lib/screens/new_student/local_widget/student_form.dart

```diff
@@ -0,0 +1,210 @@
+      child: Container(
+        child: Form(
+          key: _formKey,
+          child: SafeArea(
+            child: Column(
+              crossAxisAlignment: CrossAxisAlignment.stretch,
+              mainAxisAlignment: MainAxisAlignment.spaceEvenly,
+              mainAxisSize: MainAxisSize.max,
+              children: <Widget>[
+                Container(
+                  decoration: BoxDecoration(
+                    borderRadius: BorderRadius.circular(10.0),
+                    color: Colors.white,
+                    boxShadow: [
+                      BoxShadow(
+                        color: Colors.black,
+                        blurRadius: 2.0,
+                        spreadRadius: 0.0,
+                        offset: Offset(2.0, 2.0),
+                      ),
+                    ],
+                  ),
+                  child: Padding(
+                    padding: EdgeInsets.symmetric(
+                      horizontal: 20.0,
+                      vertical: 5.0,
+                    ),
+                    child: Column(
+                      mainAxisAlignment: MainAxisAlignment.spaceEvenly,
+                      children: [
+                        _createnameField(student),
+                        _createsurnameField(student),
+                        _createemailField(student),
+                        _createpasswordField(student),
+                        _createDescriptionField(student),
+                      ],
+                    ),
+                  ),
+                ),
+                Padding(
+                  padding: const EdgeInsets.symmetric(vertical: 16.0),
+                  child: RaisedButton(
+                    color: Colors.grey[600],
+                    onPressed: () {
+                      // Validate will return true if the form is valid, or false if
+                      // the form is invalid.
+                      if (_formKey.currentState.validate()) {
+                        widget.changeStep(false);
+                        
+                      }
+                    },
+                    child: Text(
+                      'Continue',
+                      style: TextStyle(color: Colors.white),
+                    ),
+                  ),
+                ),
+              ],
+            ),
+          ),
+        ),
+      ),
+    );
+  }
+
+  Widget _createnameField(Student student) {
+    return TextFormField(
+      decoration: const InputDecoration(
+        hintText: 'Student name',
+      ),
+      initialValue: student.name ?? '',
+      onChanged: (value) {
+        setState(() {
+          student.name = value;
+        });
+      },
+      validator: (value) {
+        if (value.isEmpty) {
+          return 'Please enter a student name';
+        }
+        return null;
+      },
+    );
+  }
+
+Widget _createsurnameField(Student student) {
+    return TextFormField(
+      decoration: const InputDecoration(
+        hintText: 'Student surname',
```

Comment:
From Figma, this should be just "Surname"

Reply:
Done